src/lib/units/day-of-week.js

```diff
@@ -99,17 +99,26 @@ function parseIsoWeekday(input, locale) {
 
 export var defaultLocaleWeekdays = 'Sunday_Monday_Tuesday_Wednesday_Thursday_Friday_Saturday'.split('_');
 export function localeWeekdays (m, format) {
+    if (typeof m === 'undefined') {
```

Comment:
There's a [utils.isUndefined](https://github.com/moment/moment/blob/develop/src/lib/utils/is-undefined.js). I forget what advantage it has, but we should be consistent.